en/admin/control.rst

```diff
@@ -2142,3 +2162,358 @@ The following shows how to use the CUBRID Manager (hereafter, CM) Administrator
+
+    % java -version Java(TM) SE Runtime Environment (build 1.6.0_05-b13)
+    Java HotSpot(TM) 64-Bit Server VM (build 10.0-b19, mixed mode)
+
+**Windows Environment**
+
+For Windows, CUBRID loads the **jvm.dll** file to run the Java Virtual Machine. CUBRID first locates the **jvm.dll** file from the **PATH** environment variable and then loads it. If it cannot find the file, it uses the Java runtime information registered in the system registry.
+
+You can configure the **JAVA_HOME** environment variable and add the directory in which the Java executable file is located to **Path**, by executing the command as follows: For information on configuring environment variables using GUI, see Installing and Configuring JDBC.
+
+*   An example of installing 64 Bit JDK 1.6 and configuring the environment variables ::
+
+    % set JAVA_HOME=C:\jdk1.6.0
+    % set PATH=%PATH%;%JAVA_HOME%\jre\bin\server
+
+*   An example of installing 32 Bit JDK 1.6 and configuring the environment variables ::
+
+    % set JAVA_HOME=C:\jdk1.6.0
+    % set PATH=%PATH%;%JAVA_HOME%\jre\bin\client
+
+If you want to specify the path of Java Virtual Machine (JVM) explicitly including cases to use other vendor's implementation instead of Sun's JVM, add the path of the **jvm.dll** file to the **JVM_PATH** variable during the installation.
+CUBRID first looks for the **jvm.dll** file in the **JVM_PATH** variable. if **JVM_PATH** is not set or if the file cannot be loaded, it looks for the file in the **JAVA_HOME** variable as described above.
+
+*   An example of configuring the **JVM_PATH** environment variable ::
+    
+    % set JVM_PATH=C:\jdk1.6.0\jre\bin\server\libjvm.dll
+
+**Linux/UNIX Environment**
+
+For Linux/UNIX environment, CUBRID loads the **libjvm.so** file to run the Java Virtual Machine. CUBRID first locates the **libjvm.so** file from the **LD_LIBRARY_PATH** environment variable and then loads it. If it cannot find the file, it uses the **JAVA_HOME** environment variable. For Linux, glibc 2.3.4 or later versions are supported. The following example shows how to configure the Linux environment variable (e.g., **.profile**, **.cshrc**, **.bashrc**, **.bash_profile**, etc.).
+
+*   An example of installing 64 Bit JDK 1.6 and configuring the environment variables in a bash shell ::
+
+    % JAVA_HOME=/usr/java/jdk1.6.0_10
+    % LD_LIBRARY_PATH=$JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server:$LD_LIBRARY_PATH
+    % export JAVA_HOME
+    % export LD_LIBRARY_PATH
+
+*   An example of installing 32 Bit JDK 1.6 and configuring the environment variables in a bash shell ::
+
+    % JAVA_HOME=/usr/java/jdk1.6.0_10
+    % LD_LIBRARY_PATH=$JAVA_HOME/jre/lib/i386/:$JAVA_HOME/jre/lib/i386/client:$LD_LIBRARY_PATH
+    % export JAVA_HOME
+    % export LD_LIBRARY_PATH
+
+*   An example of installing 64 Bit JDK 1.6 and configuring the environment variables in a csh ::
```

Comment:
propoe: in a csh --> in a csh shell as other described.

Reply:
Thank you. I appended 'shell'. but 'similar with others' seems redundant.